dbms/src/Flash/Coprocessor/DAGContext.h

```diff
@@ -316,6 +317,8 @@ class DAGContext
     /// thus we need to pay attention to scan_context_map usage that time.
     std::unordered_map<String, DM::ScanContextPtr> scan_context_map;
 
+    RuntimeFilterMgr runtime_filter_mgr;
```

Comment:
Consider to move it to a global place like MPPTaskManager and use a query_id + rf_id as key, so that global runtime filter could also use this?

Reply:
I haven't finalized the design of global yet. So this PR will not move for now. Wait until the global is confirmed to see whether to remove or keep it in later PR.
(The main reason is that the top node responsible for global may not necessarily be inside tiflash）

Reply:
I mean even top node not inside tiflash, each tiflash instance could maintain a global RuntimeFilterManager to dispatch/maintain RFs. 
Anyway, could consider this when implement global RF later.

Reply:
Yes, I will consider this when I implement global RF later.
(For local, because the responsibility of the ```RuntimeFilterMgr``` is only used in the compile phase. So it's ok to put it anywhere.)